libs/blocks/text/text.js

```diff
@@ -76,7 +76,21 @@ function decorateLinkFarms(el) {
   });
 }
 
-export default function init(el) {
+export async function loadMnemonicList(foreground) {
+  try {
+    const { base } = getConfig();
+    const stylePromise = new Promise((resolve) => {
+      loadStyle(`${base}/blocks/mnemonic-list/mnemonic-list.css`, resolve);
+    });
+    const loadModule = import('../mnemonic-list/mnemonic-list.js')
+      .then(({ decorateMnemonicList }) => decorateMnemonicList(foreground));
+    await Promise.all([stylePromise, loadModule]);
```

Comment:
Any chance we can use `loadBlock` ([link](https://github.com/adobecom/milo/blob/stage/libs/utils/utils.js#L438)), since it shares a lot of the functionality added here?